src/group_theory/quotient_group.lean

```diff
@@ -90,6 +90,18 @@ begin
   rw [quotient_group.eq, one_inv, one_mul],
 end
 
+@[to_additive quotient_add_group.mk'_eq_zero_iff]
+lemma mk'_eq_zero_iff {N : subgroup G} [nN : N.normal] (x : G) :  mk' N x = 1 ↔  x ∈ N :=
+⟨λ h, by rwa [← ker_mk N, monoid_hom.mem_ker], λ h, by rwa [← ker_mk N, monoid_hom.mem_ker] at h⟩
+
+@[to_additive quotient_add_group.mk'_eq_mk'_iff]
+lemma mk'_eq_mk'_iff {N : subgroup G} [nN : N.normal] {x y : G} : mk' N x = mk' N y ↔ x / y ∈ N :=
+begin
+  change ↑x = ↑ y ↔ _,
+  rw [quotient_group.eq, ← N.inv_mem_iff, div_eq_mul_inv, subgroup.normal.mem_comm_iff nN,
+    mul_inv_rev, inv_inv],
+end
```

Comment:
A shorter proof is:
```suggestion
@[to_additive quotient_add_group.mk'_eq_mk'_iff]
lemma mk'_eq_mk'_iff {N : subgroup G} [nN : N.normal] {x y : G} : mk' N x = mk' N y ↔ x / y ∈ N :=
begin
  refine eq_comm.trans (quotient_group.eq.trans _),
  rw [nN.mem_comm_iff, div_eq_mul_inv]
end
```
Although this lemma feels like it should be stated in terms of `coe` not `mk'`:
```suggestion
@[to_additive quotient_add_group.eq_of_normal]
lemma eq_of_normal {N : subgroup G} [nN : N.normal] {x y : G} :
  (x : quotient N) = y ↔ x / y ∈ N :=
begin
  refine eq_comm.trans (quotient_group.eq.trans _),
  rw [nN.mem_comm_iff, div_eq_mul_inv]
end
```
or perhaps even in terms of `inv` not `div`:
```
@[to_additive quotient_add_group.eq_of_normal ]
lemma eq_of_normal {N : subgroup G} [nN : N.normal] {x y : G} :
  (x : quotient N) = y ↔ x * y⁻¹ ∈ N :=
eq_comm.trans (quotient_group.eq.trans nN.mem_comm_iff)
```

Reply:
I don't about ` x * y⁻¹`: the additive version will be `x + -y` that seems wrong, even if ` x * y⁻¹` seems better in the multiplicative world.